packages/builder/src/utils/wagmi.ts

```diff
@@ -4,19 +4,83 @@ import {
   injectedWallet,
   metaMaskWallet,
 } from "@rainbow-me/rainbowkit/wallets";
-import { chain, configureChains, createClient } from "wagmi";
+import { configureChains, createClient } from "wagmi";
+import {
+  fantom as fantomChain,
+  fantomTestnet as fantomTestnetChain,
+  mainnet,
+  arbitrum,
+  optimism,
+  goerli,
+  arbitrumGoerli,
+  hardhat,
+  avalancheFuji as avalancheFujiChain,
+  avalanche as avalancheChain,
+} from "wagmi/chains";
 import { alchemyProvider } from "wagmi/providers/alchemy";
 import { infuraProvider } from "wagmi/providers/infura";
 import { publicProvider } from "wagmi/providers/public";
 import PublicGoodsNetworkIcon from "common/src/icons/PublicGoodsNetwork.svg";
-import { FantomFTMLogo, FTMTestnet, OPIcon } from "../assets";
-
-const ftmTestnetIcon = FTMTestnet;
-const ftmMainnetIcon = FantomFTMLogo;
+import { polygon, polygonMumbai } from "@wagmi/core/chains";
+import { FantomFTMLogo } from "../assets";
 
 // RPC keys
-const alchemyId = process.env.REACT_APP_ALCHEMY_ID;
-const infuraId = process.env.REACT_APP_INFURA_ID;
+const alchemyId = process.env.REACT_APP_ALCHEMY_ID!;
+const infuraId = process.env.REACT_APP_INFURA_ID!;
+
+export const avalanche: Chain = {
+  ...avalancheChain,
+  rpcUrls: {
+    default: {
```

Comment:
Out of curiosity, what is the difference between `default` and `http`?

Reply:
in newer wagmi versions, this is no longer the shape, not sure what the purpose is in this one, but the default chains exported from wagmi/chains all have the same http and default.